tools/Azure.Mcp.Tools.AzureManagedLustre/tests/Azure.Mcp.Tools.AzureManagedLustre.LiveTests/AzureManagedLustreCommandTests.cs

```diff
@@ -92,5 +110,126 @@ public async Task Should_get_sku_info_no_zonal_support()
+        {
+            var fsName = $"amlfs-{Guid.NewGuid().ToString("N")[..8]}";
+            var subnetId = Environment.GetEnvironmentVariable("AMLFS_SUBNET_ID");
+            var location = Environment.GetEnvironmentVariable("LOCATION");
+
+            // Calculate CMK required variables
+
+            var keyUri = Environment.GetEnvironmentVariable("KEY_URI_WITH_VERSION");
+            var keyVaultResourceId = Environment.GetEnvironmentVariable("KEY_VAULT_RESOURCE_ID");
+            var userAssignedIdentityId = Environment.GetEnvironmentVariable("USER_ASSIGNED_IDENTITY_RESOURCE_ID");
+
+            // Calculate HSM required variables
+            var hsmDataContainerId = Environment.GetEnvironmentVariable("HSM_CONTAINER_ID");
+            var hsmLogContainerId = Environment.GetEnvironmentVariable("HSM_LOGS_CONTAINER_ID");
+
+            var result = await CallToolAsync(
+                "azmcp_azuremanagedlustre_filesystem_create",
+                new()
+                {
+                        { "subscription", Settings.SubscriptionId },
+                        { "resource-group", Settings.ResourceGroupName },
+                        { "location", location },
+                        { "name", fsName },
+                        { "sku", "AMLFS-Durable-Premium-500" },
+                        { "size", 4 },
+                        { "zone", 1 },
+                        { "subnet-id", subnetId },
+                        { "hsm-container", hsmDataContainerId },
+                        { "hsm-log-container", hsmLogContainerId },
+                        { "custom-encryption", true },
+                        { "key-url", keyUri },
+                        { "source-vault", keyVaultResourceId },
+                        { "user-assigned-identity-id", userAssignedIdentityId },
+                        { "maintenance-day", "Monday" },
+                        { "maintenance-time", "01:00" },
+                        { "root-squash-mode", "All" },
+                        { "no-squash-nid-list", "10.0.0.4"},
+                        { "squash-uid", 1000 },
+                        { "squash-gid", 1000 }
+                });
+
+            var fileSystem = result.AssertProperty("fileSystem");
+            Assert.Equal(JsonValueKind.Object, fileSystem.ValueKind);
+
+            var name = fileSystem.GetProperty("name").GetString();
+            Assert.Equal(fsName, name);
+
+            var fsLocation = fileSystem.GetProperty("location").GetString();
+            Assert.Equal(location, fsLocation);
+
+            var hasCapacity = false;
+            var capacityValue = 0;
+
+            if (fileSystem.TryGetProperty("storageCapacityTiB", out var capTiB) && capTiB.ValueKind == JsonValueKind.Number)
+            {
+                hasCapacity = true;
+                capacityValue = capTiB.GetInt32();
+            }
+            Assert.True(hasCapacity, "Expected a storage capacity property.");
+            Assert.Equal(4, capacityValue);
```

Comment:
Doesn't seem like this should be handled with `TryGetProperty`.

```suggestion
            var capacity = fileSystem.AssertProperty("storageCapacityTiB");
            Assert.Equal(JsonValueKind.Number, capacity.ValueKind);
            Assert.Equal(4, capacity.GetInt32());
```